lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -33,9 +33,10 @@ function multiplyFractions(node) {
   const numeratorArgs = [];
   const denominatorArgs = [];
   node.args.forEach(operand => {
-    if (Node.Type.isOperator(operand, '/')) {
-      numeratorArgs.push(operand.args[0]);
-      denominatorArgs.push(operand.args[1]);
+    if (Node.Type.isFraction(operand)) {
+      const fraction = Node.Type.getFraction(operand);
```

Comment:
can you do, for the case with unary minus, call getFraction recursively then negate the numerator before returning? I think that should work - how's that sound?